api/shape/define.js

```diff
@@ -14,73 +14,69 @@ export const defThreejsMaterial = (name, definition) =>
 
 export const defTool = (name, definition) => define(`tool:${name}`, definition);
 
-export const defGrblSpindle = (
-  name,
-  { cutDepth = 0.2, rpm, feedRate, drillRate, diameter, jumpZ = 1 } = {}
-) =>
-  defTool(name, {
-    grbl: {
-      type: 'spindle',
-      cutDepth,
-      cutSpeed: rpm,
-      feedRate,
-      drillRate,
-      diameter,
-      jumpZ,
-    },
-  });
+export const GrblSpindle = ({
+  cutDepth = 0.2,
+  rpm,
+  feedRate,
+  drillRate,
+  diameter,
+  jumpZ = 1,
+} = {}) => ({
+  grbl: {
+    type: 'spindle',
+    cutDepth,
+    cutSpeed: rpm,
+    feedRate,
+    drillRate,
+    diameter,
+    jumpZ,
+  },
+});
 
-export const defGrblDynamicLaser = (
-  name,
-  {
-    cutDepth = 0.2,
-    diameter = 0.09,
-    jumpPower = 0,
-    power = 1000,
-    speed = 1000,
+export const GrblDynamicLaser = ({
```

Comment:
Are you thinking about doing some laser stuff?

Reply:
Laser stuff was working ok, but this is just clean up.

I'll get the laser set up in a while and make sure it's working with the simplified system.